### DISCUSSION TITLE
can someone clarify the "combine images" function?

### DISCUSSION THREAD

**Discussion by dangerweenie** [thygate/stable-diffusion-webui-depthmap-script#144] [Q&A] [answered]
> When Combine into one image is enabled, the depthmap will be combined with the original image, the orientation can be selected with Combine axis. When disabled, the depthmap will be saved as a 16 bit single channel PNG as opposed to a three channel (RGB), 8 bit per channel image when the option is enabled.

I'm not quite sure what this means - why would someone merge RGB + Z?  Also not sure about the axis parameter...

**Comment by thygate** [comment, marked as answer]
Looking Glass Studio uses a RGB + depth Side-by-side image input to create the 'Quilt' files required for viewing on their holographic displays.
This was my original intent for the script, viewing sd generated content in realtime on a [looking glass portrait](https://lookingglassfactory.com/looking-glass-portrait) device. 
As explained in the readme, if you want the highest resolution depthmap, save as PNG, and don't combine.
The axis is the stacking direction, horizontal, or vertical (eg side-by-side or under-over).

**Reply by dangerweenie** [reply]
Thanks for the clarification 🙏
